### PR TITLE
SSE-2869   Review and Update WAF Rules for Load Balancer

### PR DESCRIPTION
**Pull request by rafal-surowiec-co**

### CODE REVIEW

File: infrastructure/frontend/frontend.template.yml [hunk 1]
```diff
@@ -282,6 +292,134 @@ Resources:
...
       ToPort: !Ref ContainerPort
 
+  FrontendWebAcl:
+    Type: 'AWS::WAFv2::WebACL'
+    Properties:
+      Name: !Sub '${AWS::StackName}-${Environment}-acl'
+      Scope: REGIONAL
+      Description: Web ACL for Application Load Balancer
+      DefaultAction:
+        Allow: {}
```
Comment (abdul-gds) [inline infrastructure/frontend/frontend.template.yml:302]: value required

**Review by keith-hart — CHANGES_REQUESTED** [review]
shouldn't lines 99-120 be changed now the policy character limit is changed

**Review by MartinH-DI — DISMISSED** [review]
Looks good

**Comment by rafal-surowiec-co** [comment]
> shouldn't lines 99-120 be changed now the policy character limit is changed

Addressed

**Pull request closed without merging**